contrib/go-audit.init

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+#
+# go-audit
+#
+# chkconfig:   - 15 85
+# description:  go-audit is an alternative to the auditd daemon that \
+#               ships with many distros.
+# processname: go-audit
+# config:      /etc/go-audit.yaml
+# pidfile:     /var/run/go-audit.pid
+
+# Source function library.
+. /etc/rc.d/init.d/functions
+
+exec="/usr/bin/go-audit"
+prog=$(basename $exec)
+pidfile=/var/run/${prog}.pid
+
+CONFIG_FILE="/etc/go-audit.yaml"
+LOG_DIRECTORY="/var/log/go-audit"
+
+[ -e /etc/sysconfig/$prog ] && . /etc/sysconfig/$prog
+
+start() {
+    [ -f $pidfile ] && checkpid $(cat $pidfile) && echo "$prog already running" && return
+
+    echo -n $"Starting $prog: "
+    nohup $exec --config $CONFIG_FILE >>${LOG_DIRECTORY}/go-audit.out 2>>${LOG_DIRECTORY}/go-audit.err </dev/null &
```

Comment:
`nohup` blocks SIGHUP which is used while reloading services, so i would like to suggest to use `daemonize` instead.